WalletWasabi/Crypto/ZeroKnowledge/CredentialPresentation.cs

```diff
@@ -0,0 +1,37 @@
+﻿using NBitcoin.Secp256k1;
+using WalletWasabi.Crypto.Groups;
+
+namespace WalletWasabi.Crypto.ZeroKnowledge
+{
+	public class CredentialPresentation
+	{
+		public CredentialPresentation(GroupElement Ca, GroupElement Cx0, GroupElement Cx1, GroupElement CV, GroupElement S)
```

Comment:
Is there some resource where one can find out what these variables denote?

Reply:
in the [paper](https://github.com/zkSNACKs/WabiSabi) (direct [PDF link](https://github.com/zkSNACKs/WabiSabi/releases/latest/download/WabiSabi.pdf)) section 4.

Ca is a randomized commitment to the amount attribute, Cx0, Cx1 and CV are randomized commitments relating to the algebraic MAC, and S is the serial number for the credential used to prevent double spending